values.yaml

```
@@ -72,7 +72,20 @@ taiga:
   # emailSmtpPort: 587
   # emailSmtpUser: "user@example.com"
   # emailSmtpPassword: ""
-
+  ldap:
+    Enabled: False
```

The rest of the values.yaml file has `enabled` as fully lowercase, so it'd be nicer to make it more uniform
```suggestion
    enabled: False
```